api/services/recipeform.service.js

```diff
@@ -0,0 +1,121 @@
+const measures = require('../measurements');
+
+const toMap = measures.measuresToMap;
+const toFind = measures.measuresToFind;
+
+exports.buildUrl = (name) => {
+    return name.toLowerCase().replace(/\s/g, '-');
+}
+
+//replace/update when tags library is found
+exports.formatKeywords = (keywords) => {
+    let formattedKeywords = keywords.replace(',', '').split(' ');
+    return formattedKeywords;
+}
+
+exports.formatDirections = (directions) => {
+    let formattedDirections = directions.split('\n');
+    return formattedDirections;
+}
+
+exports.formatIngredients = (ingredients) => {
+    const ingredientArray = ingredients.split("\n");
```

Comment:
Double quotes here instead of single. This is something you can change in `prettier` settings to always use one or the other